routes/router.js

```diff
@@ -0,0 +1,15 @@
+const api = require('../api');
+
+let initializeRouter = (app) => {
+    app.get('/', (request, response) => {
+        response.send('Welcome to Canvas Consumer API.');
+    });
+    app.post('/link-account', api.linkAccount);
```

Comment:
If we keep every routing-logic here, it's gonna be a huge file in the end. How about moving all the app.post('link-ccount') etc inside the api? And just use courseApi.init(app) or something here. Then we have control over the API-path in the api-files as well.